katalog/dual-nginx/external/patch/cm.yml

```diff
@@ -0,0 +1,4 @@
+- op: "replace"
+  path: "/metadata/labels"
+  value:
+    app: ingress-external
```

Comment:
Why do we need to patch the `labels`? Can't we use Kustomize's `commonLabels` feature which takes care of updating `selector`s too?

Reply:
You're right. Moved ;)